yocto-builder/Dockerfile

```diff
@@ -22,7 +23,9 @@ RUN apt-get update \
       wget \
       xterm \
     && apt-get autoremove \
-    && apt-get clean
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+RUN pip3 install kas
```

Comment:
is that tool used by anyone in yocto-builder? should be some information available in the readme?

Reply:
What do you expect exactly?

Reply:
I understood you to say that the kas tool still needs to be configured. Or can it be that this PR is only used to install the tool?

Reply:
`kas` is only added in this PR... 
The configuration of a yocto image is project specific. We could add an example how to use kas in context of bb-yocto-builder but this is not refined. 
For me using `kas` here, e.g. in `pipeline.yaml` is another, not written yet, user story. Would we describe the current bb-yocto-builder way and an alternative way via `kas`? Or will `kas` replace the current implementation? Currently we didn't really verify `kas`. 

@JanZachmann Your opinion?

Reply:
Changed my mind. I'll use `kas` in `bb-yocto-builder` as well.

Reply:
done